small/small.c

```diff
@@ -202,15 +299,15 @@ void
 smfree(struct small_alloc *alloc, void *ptr, size_t size)
 {
 	struct mempool *pool = mempool_find(alloc, size);
-	if (pool == NULL) {
+	if (unlikely(pool == NULL)) {
```

Comment:
That's an optimization and is not relevant to current commit. Please move it to separate commit.